Add server-community, puppet-enterprise and github-actions files. (#110)
odidev/ecosystem-dashboard-for-arm: content/commercial_packages/gitHub-actions.md

---
name: GitHub Actions Self-hosted Runner
vendor: GitHub
category: DevOps

> description: A self-hosted runner is a system that you deploy and manage to execute jobs from GitHub Actions on GitHub.com.Self-hosted runners can be physical, virtual, in a container, on-premises, or in a cloud.

Author: A self-hosted runner is a system that you deploy and manage to execute jobs from GitHub Actions on GitHub.com.Self-hosted runners can be physical, virtual, in a container, on-premises, or in a cloud.
```suggestion
description: A self-hosted runner is a system that deploys and manages the execution of  jobs from GitHub Actions on GitHub.com. Self-hosted runners can be physical, virtual, in a container, on-premises, or in a cloud.
```

> product_url: https://docs.github.com/en

Author: I think it should be: https://docs.github.com/en/actions

works_on_arm: true
release_date_on_arm: 12/11/2019

optional_info:

>     homepage_url: https://docs.github.com/en

Author: I think it should be: https://docs.github.com/en/actions
